sqladmin/forms.py

```diff
@@ -337,7 +337,15 @@ def conv_text(
     def conv_boolean(
         self, model: type, prop: ColumnProperty, kwargs: Dict[str, Any]
     ) -> UnboundField:
-        return BooleanField(**kwargs)
+        if not prop.columns[0].nullable:
+            kwargs.setdefault("render_kw", {})
+            kwargs["render_kw"]["class"] = "form-check-input"
+            return BooleanField(**kwargs)
+
+        kwargs["allow_blank"] = True
+        kwargs["choices"] = [(True, "True"), (False, "False")]
+        kwargs["coerce"] = lambda v: v == "True"
+        return SelectField(**kwargs)
```

Comment:
It's strange that selecting `None` and `False` from UI work fine and as expected.
But selecting `True` gives `Not a valid choice`.